Easydict/NewApp/View/SettingView/Tabs/DisabledAppTab.swift

```diff
@@ -0,0 +1,230 @@
+                allowedContentTypes: [.application],
+                allowsMultipleSelection: true
+            ) { result in
+                switch result {
+                case let .success(urls):
+                    disabledAppViewModel.newAppURLsSelected(from: urls)
+                case let .failure(error):
+                    print("fileImporter error: \(error)")
+                    disabledAppViewModel.isShowImportErrorAlert.toggle()
+                }
+            }
+            .alert(isPresented: $disabledAppViewModel.isShowImportErrorAlert) {
+                Alert(title: Text(""), message: Text("setting.disabled.import_app_error.message"), dismissButton: .default(Text("ok")))
+            }
+    }
+
+    var appListView: some View {
+        List(selection: $disabledAppViewModel.selectedAppModels) {
+            ForEach(disabledAppViewModel.appModelList, id: \.self) { app in
+                BlockAppItemView(with: app)
+                    .tag(app)
+            }
+            .listRowSeparator(.hidden)
+        }
+        .listStyle(.plain)
+        .scrollIndicators(.never)
+    }
+
+    var appListViewWithToolbar: some View {
+        VStack(spacing: 0) {
+            appListView
+
+            listToolbar
+        }
+        .clipShape(RoundedRectangle(cornerRadius: 8))
+        .overlay(content: {
+            RoundedRectangle(cornerRadius: 8)
+                .stroke(Color("list_border_color"), lineWidth: 0.5)
+        })
+        .padding(.horizontal, 25)
+        .padding(.bottom, 25)
+        .onTapGesture {
+            disabledAppViewModel.selectedAppModels = []
+        }
+    }
+
+    var body: some View {
+        VStack {
+            Text("disabled_title")
+                .padding(.horizontal)
+                .padding(.top, 18)
+                .padding(.bottom, 8)
+
+            appListViewWithToolbar
+        }
+        .environmentObject(disabledAppViewModel)
+    }
+}
+
+@available(macOS 13.0, *)
+private struct ListToolbar: View {
+    @EnvironmentObject private var disabledAppViewModel: DisabledAppViewModel
+
+    var body: some View {
+        VStack(spacing: 0) {
+            Divider()
+            HStack(spacing: 0) {
+                ListButton(imageName: "plus", enabled: true) {
+                    disabledAppViewModel.isImporting.toggle()
+                }
+                Divider().padding(.vertical, 1)
+                let isNoSelectedApps = disabledAppViewModel.selectedAppModels.isEmpty
+                ListButton(imageName: "minus", enabled: !isNoSelectedApps) {
+                    disabledAppViewModel.removeDisabledApp()
+                }
+                .disabled(isNoSelectedApps)
+                Spacer()
+            }
+            .padding(2)
+        }
+        .frame(height: 28)
+        .background(Color("add_minus_bg_color"))
+    }
+}
+
+@available(macOS 13.0, *)
+private struct ListButton: View {
+    var imageName: String
```

Comment:
Better named as `systemName` just like the `Image`'s parameter, otherwise it may be confusing.